scripts/linz-bde-schema-load.bash

```diff
@@ -84,21 +102,24 @@ dbpatch-loader - fake 2>&1 | grep -q "database.*does not exist" &&
     DBPATCH_SUPPORTS_STDOUT=no ||
     DBPATCH_SUPPORTS_STDOUT=yes
 
-if test $PGDATABASE = "-" -a $DBPATCH_SUPPORTS_STDOUT != yes; then
+if test "$PGDATABASE" = "-" -a $DBPATCH_SUPPORTS_STDOUT != yes
```

Comment:
`$DBPATCH_SUPPORTS_STDOUT` should also be quoted. If it's not, and the value is ever empty, then the expression gets borked. Ditto for other unquoted variable references.

Reply:
Does this apply only when the '$' appears at the beginning of the variable or every occurrence?

Reply:
e.g should DBPATCH_SUPPORTS_STDOUT on lines 102/103 be quoted?

Reply:
Lines 102/103 are *assignments.* You only need to quote variable *references.*